Modules/Sources/NetworkingCore/Remote/OrdersRemote.swift

```diff
@@ -598,6 +598,24 @@ public extension OrdersRemote {
         case customerID
         case currency
     }
+
+    /// Loads a single order asynchronously for POS
+    /// - Parameters:
+    ///   - siteID: Site for which we'll fetch the order.
+    ///   - orderID: ID of the order to load.
+    /// - Returns: The loaded Order.
+    /// - Throws: Network or parsing errors.
+    func loadPOSOrder(siteID: Int64, orderID: Int64) async throws -> Order {
```

Comment:
Question: This could be an async wrapper for existing loadOrder, right? So no need to define the function specifically for POS orders

Reply:
I first reused the existing method, but it's a bit awkward since it uses non-async version of `enqueue` that returns an optional `Data?` and optional `Error? which requires to set some `unknown` error state to make sure all the code paths are covered. Seemed cleaner to create this new async method.

Reply:
Gotcha, sounds good to me! 🕺